### DISCUSSION TITLE
What name should be used for you all?

### DISCUSSION THREAD

**Discussion by clawsoon** [tskit-dev/tskit#1665] [Q&A] [answered]
If I use data from the Unified Genealogy paper, what collective noun should I use for you all?  In other words, what's the fill-in-the-blank for this paragraph?

"These genetic variants were collected by the Simons Genome Diversity Project, the Human Genome Diversity Project, and the 1000 Genomes Project.  Dates were estimated by ____________________."

This is for subtitles on a video.  I'm planning to put full references and links into the end credits, but for the subtitles themselves I'll only have room for a few words.

Thanks!

**Comment by jeromekelleher** [comment, marked as answer]
I think for the unified genealogy paper saying the dates were estimated by "Wohns et al" is probably the most appropriate, since it's not something we did as a community. It's hard to know where to draw the line though!

**Reply by clawsoon** [reply]
Thanks, I'll use that.
